app/views/suggested_edit/category_index.html.erb

```diff
@@ -35,7 +35,11 @@
     </tr>
     <% categories.each do |cat| %>
       <% next if (cat.min_view_trust_level || -1) > (current_user&.trust_level || 0) %>
-      <% sug_edits = SuggestedEdit.where(post: Post.undeleted.in(cat), active: !@show_decided).count %>
```

Comment:
I don't know if this edit-counting code should really be in the view, but that's where I found it so I expanded it here at least for now.